### PR TITLE
Maven dependencies

### PR DESCRIPTION
**Pull request by smolamarcin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.epam</groupId>
+    <artifactId>battleship</artifactId>
+    <version>1.0-SNAPSHOT</version>
```
Comment (rivalez) [inline pom.xml]: please provide version 1.0.1 as stable one

File: src/main/java/com/epam/App.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.epam;
+
+public class App {
```
Comment (rivalez) [inline src/main/java/com/epam/App.java]: rename package to solid.nie and class name to Game

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>findbugs-maven-plugin</artifactId>
+                <version>3.0.4</version>
+                <configuration>
+                    <xmlOutput>true</xmlOutput>
+                    <!-- Optional directory to put findbugs xdoc xml report -->
```
Comment (rivalez) [inline pom.xml]: please delete comment

File: src/main/java/com/epam/App.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.epam;
```
Comment (Ozymadias) [inline src/main/java/com/epam/App.java]: com.epam.solidnie

**Comment by smolamarcin** [comment]
only pom.xml

**Pull request merged**